src/psyclone/tests/test_files/gocean1p0/reordered_dependent_invoke.f90

```diff
@@ -0,0 +1,52 @@
+!-------------------------------------------------------------------------------
```

Comment:
Please could you use the current license header here.

Reply:
I think this reordered one is a test I made to try some coverage improvements, but actually I can remove.